samples/sample_fei/src/fei_encode.cpp

```diff
@@ -22,6 +22,7 @@ or https://software.intel.com/en-us/media-client-solutions-support.
 #error MFX_VERSION not defined
 #endif
 
+
```

Comment:
please, do not commit files without any real changes. These are leftovers from your previous changes. Let's reduce delta

Reply:
Removed